src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -134,6 +134,7 @@ def load_command_table(self, _):
         g.custom_command('start', 'aks_agentpool_start', supports_no_wait=True)
         g.wait_command('wait')
         g.custom_command('operation-abort', 'aks_agentpool_operation_abort', supports_no_wait=True)
+        g.custom_command("delete-machines", "aks_agentpool_delete_machines", supports_no_wait=True)
```

Comment:
> -  FAIL - HIGH severity: missing_command_test_coverage
>     Repo: ./, Src Branch: HEAD, Target Branch: origin/dev - Failed.
> 	Missing command test coverage: `aks nodepool "delete-machines"`
> 	Please add some scenario tests for the new command
> 	Or add the command with missing_command_test_coverage rule in linter_exclusions.yml

Rules in CI might be a bit rigid, I guess changing `"delete-machines"` to `'delete-machines'` would fix the issue.

Reply:
done